apollo-router/src/axum_factory/tests.rs

```diff
@@ -795,11 +795,6 @@ async fn response_with_custom_endpoint_wildcard() -> Result<(), ApolloRouterErro
 
 #[tokio::test]
 async fn it_extracts_query_and_operation_name_on_get_requests() -> Result<(), ApolloRouterError> {
-    // TODO re-enable after the release
```

Comment:
Have we decided this test is redundant or doesn't work or ...?

Reply:
I don't know why we disabled it. All I know is it's been a while since we have this TODO :p and I don't think we will re-enable it because we can live without it. But I might be wrong :p

Reply:
You are right, the issue happened because we were trying to set several global subscribers. Having only one test that checks for spans is enough, which is accomplished by previous tests. I think I removed this todo when i was working on custom endpoints so it's prolly a failed conflict merge at this point.

Reply:
Ok.